src/components/recipe-form.js

```diff
@@ -183,6 +185,13 @@ const RecipeForm = ({
     await uploadImageToS3(image, submitMutation)
   }
 
+  if (!isOwner)
+    return (
+      <div sx={{ textAlign: `center` }}>
+        Woops, looks like you aren't the owner of this recipe!
+      </div>
+    )
+
```

Comment:
I'm getting this for all the different instances of the form (create, edit, variant). The code looks right so I dunno what's going on there.